bookkeeper-server/src/test/java/org/apache/bookkeeper/tls/TestTLS.java

```diff
@@ -635,6 +635,47 @@ public void testBookieAuthPluginRequireClientTLSAuthenticationLocal() throws Exc
         assertTrue(cert instanceof X509Certificate);
     }
 
+    /**
+     * Verify that given role in client certificate is checked when BookieAuthZFactory is set.
+     * Positive test case where all given roles are present.
```

Comment:
this test does look like a "Positive test"
you are asserting that the client is not able to work

Reply:
If `testClient()` fails, it will be because of an exception thrown during Authorization.
And hence we catch that and fail the test.
Otherwise, if the test passes, it never reaches the `catch` block and finishes, passing the test
Sorry for the confusing comment, I can change that in a follow up.

Reply:
Fixed in [7276947](https://github.com/apache/bookkeeper/commit/72769477f52e4982081ae87a8e43da0a0e34e2d1)